felix/statusrep/status_reporter.go

```diff
@@ -182,6 +182,8 @@ loop:
 					Hostname:   esr.hostname,
 					EndpointID: msg.Id.EndpointId,
 				}
+			case *proto.DataplaneInSync:
```

Comment:
Could we remove the old in-sync channel completely now?  This should be better.

Reply:
I guess so, but avoided out of nerves of breaking something else :laughing: WIll give it a shot

Reply:
Done - will monitor CI in-case I broke anything else :laughing: